.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,189 @@
+          pip3 install --user "conan==${{ env.CONAN_VERSION }}"
+        shell: pwsh
+
+      - name: Setup Conan
+        if: startsWith(matrix.os, 'windows')
+        run: |
+          conan --version
+          conan profile detect -f
+
+      - name: Install dependencies
+        if: startsWith(matrix.os, 'windows')
+        run: |
+          mkdir build-test
+          cd build-test
+          conan install .. `
+              -s build_type=RelWithDebInfo `
+              -c tools.cmake.cmaketoolchain:generator="NMake Makefiles" `
+              -b missing `
+              --output-folder=. `
+              -o with_openssl3="$Env:C2_USE_OPENSSL3"
+        shell: powershell
+
+      - name: Build
+        if: startsWith(matrix.os, 'windows')
+        shell: pwsh
+        env:
+          # Enable PCH on Windows, later we can have it depend on the run/Qt Version like in the build workflow, I just want to make it work
+          C2_WINDOWS_USE_PCH: True
+        run: |
+          cmake `
+              -G"NMake Makefiles" `
+              -DCMAKE_BUILD_TYPE=RelWithDebInfo `
+              -DBUILD_TESTS=On `
+              -DBUILD_APP=OFF `
+              -DCMAKE_TOOLCHAIN_FILE="conan_toolchain.cmake" `
+              -DUSE_PRECOMPILED_HEADERS=${{ env.C2_WINDOWS_USE_PCH }} `
+              -DBUILD_WITH_CRASHPAD="$Env:C2_ENABLE_CRASHPAD" `
+              -DCHATTERINO_LTO="$Env:C2_ENABLE_LTO" `
+              -DCHATTERINO_PLUGINS="$Env:C2_PLUGINS" `
+              -DBUILD_WITH_QT6="$Env:C2_BUILD_WITH_QT6" `
+              ..
+          set cl=/MP
+          nmake /S /NOLOGO
+        working-directory: build-test
+
+      - name: Download and extract Twitch PubSub Server Test
+        if: startsWith(matrix.os, 'windows')
+        run: |
+          mkdir pubsub-server-test
+          Invoke-WebRequest -Uri "https://github.com/Chatterino/twitch-pubsub-server-test/releases/download/v1.0.7/server-v1.0.7-windows-amd64.zip" -outfile "pubsub-server.zip"
+          Expand-Archive pubsub-server.zip -DestinationPath pubsub-server-test
+          rm pubsub-server.zip
+        shell: pwsh
+
+      - name: Checkout httpbox
+        if: startsWith(matrix.os, 'windows')
+        uses: actions/checkout@v4
+        with:
+          repository: "kevinastone/httpbox"
+          path: "httpbox"
+
+      - name: Test
+        if: startsWith(matrix.os, 'windows')
+        timeout-minutes: 30
+        run: |
+          cd ..\httpbox
+          cargo build
+          cargo run -- --port 9051 &
+          cd ..\build-test
+          cd ..\pubsub-server-test
+          Invoke-WebRequest -Uri "https://github.com/Chatterino/twitch-pubsub-server-test/raw/master/cmd/server/server.crt" -outfile "server.crt"
+          Invoke-WebRequest -Uri "https://github.com/Chatterino/twitch-pubsub-server-test/raw/master/cmd/server/server.key" -outfile "server.key"
```

Comment:
This could use the same tag that _Download and extract Twitch PubSub Server Test_ uses, which could be defined at the top like `QT_QPA_PLATFORM`.